content/app_competition_rules.tex

```diff
@@ -71,17 +85,123 @@ \subsection{Winner and Rankings}
 
 \end{enumerate}
 
-\subsection{Champions Cup and Challenge Shield}
-\label{sec:twoCompetitions}
-In order to provide better matched games for teams of all abilities the RoboCup Standard Platform League shall be divided into two separate competitions: the Champions Cup for the strongest teams and the Challenge Shield for all other teams. Final assignment of teams to each competition occurs at RoboCup based on initial game performance.
+\subsubsection{Promotion Games}
+
+The bottom 2 teams of each group of the Champions Cup after the round-robin, along with all semi-finalist teams of the Challenge Shield knock-out competition proceed to the promotion games.
+
+There will be 4 promotion games. By random draw, each game will have one team from the Champions Cup and one team from the Challenge Shield. The winning team of the promotion game will be guaranteed a spot in the Champions Cup in the next RoboCup tournament. The loosing team of the promotion game, may still be placed in the Champions Cup in the next RoboCup tournament subject to available places and their Glicko ranking.
+
+
+\subsubsection{Champions Cup selection}
+
+Teams are selected for the Champions Cup as follows until all places are filled:
+\begin{enumerate}
+    \item All qualified teams that made the knockout stage of the previous year's Champions Cup.
+    \item All qualified teams that won the promotion game  of the previous year's Champions Cup.
+    \item The highest Glicko ranked qualified team, until all remaining places are filled.
+\end{enumerate}
+
+All remaining qualified teams are then selected for the Challenge Shield.
+
+
+\subsection{Challenge Shield Rule Changes}
+\label{sec:cs-rule-changes}
+
+To encourage new teams to join the SPL and reduce the bar to entry, the Challenge Shield will play under a modified rule-set to the competition rules (Sections \ref{sec:game_process} and \ref{sec:forbidden_act}). The intent of these rule modifications is to phrase a distinguishing factor between the Champions Cup and Challenge Shield as a higher level of refinement of a team's autonomous robot software through:
+\begin{itemize}
+    \item Reducing the harshness of penalties to allow robots to be on the field for a greater portion of the game time.
+    \item Give robots more time to react to different game circumstances.
+    \item Keeping consistency with the Champions Cup rules for teams that are promoted.
```

Comment:
Do I understand it correctly that promotion games are played according to champions cup rules?

Reply:
This I haven't fully decided yet, and are open to discussion. I think the promotion games give encouragement to both the CS and CC teams involved. I would probably propose holding it under the CS rule-set as it is a relaxed and more flexible rule set. I don't think it would put the CC teams at a disadvantage - if anything their "harder" ruleset would give them an advantage towards more reliable and quicker play. Where as the other way around significantly disadvantages the CS teams.